docs/src/content/docs/openapi-fetch/examples.md

```diff
@@ -68,194 +68,24 @@ client.GET("/some-authenticated-url", {
 });
 ```
 
-## Caching
```

Comment:
This may be slightly-controversial, but I removed the section on caching. I have a feeling like encouraging users to roll their own caching strategy in general is a bad idea. This should be replaced with more framework-specific examples that encourage good habits within your reactivity framework.

If a user is dedicated rolling their own client-side caching, then they probably aren’t the type to follow instructions 😄